.woodpecker/test.yaml

```diff
@@ -40,6 +40,7 @@ steps:
       - go run go.woodpecker-ci.org/woodpecker/v2/cmd/cli lint
     environment:
       WOODPECKER_DISABLE_UPDATE_CHECK: true
+      WOODPECKER_WARNING_AS_ERROR: true
```

Comment:
As this is an explicit linter config, can we rename it to `WOODPECKER_LINT_STRICT´?

Reply:
sure, i have no clear preferences

Reply:
5547e0cdb784514b673e9c90aa36a230a8166f77

Reply:
Thanks!